package.json

```diff
@@ -3,7 +3,10 @@
   "version": "0.15.5",
   "description": "The Phoenix LiveView JavaScript client.",
   "license": "MIT",
-  "main": "./priv/static/phoenix_live_view.js",
+  "module": "./priv/static/phoenix_live_view.js",
+  "exports": {
+    "import": "./priv/static/phoenix_live_view.js"
+  },
```

Comment:
Isn't this still going to break phoenix_live_view for everyone out there today because of npm file dependencies?

Reply:
I think the opposite - we were having issues with snowpack because we weren't doing this. This is telling builders that when importing phoenix_live_view to use that built file which should have the broadest compatibility.

I tested with livebook and a freshly generated project and it worked correctly. Will double check again!